docs/design/add-in-commands.md

```diff
@@ -23,13 +24,20 @@ Add-in commands are UI elements that extend the Office UI and start actions in y
 
 ![Screenshot showing add-in commands in Excel on the web.](../images/add-in-commands-2.png)
 
+## Types of add-in commands
+
+There are two types of add-in commands, based on the kind of action that the command triggers:
```

Comment:
As per the MSFT style guide, if the lead-in sentence for a list is a complete sentence, use a period instead of a colon.
```suggestion
There are two types of add-in commands, based on the kind of action that the command triggers.
```